src/pyhf/probability.py

```diff
@@ -40,6 +41,17 @@ def log_prob(self, value):
         return tensorlib.sum(_log_prob, axis=-1)
 
 
+class Simultaneous(object):
```

Comment:
This class should also have docstrings, but we already agreed this would get put in on the toys PR.

Reply:
ok